WalletWasabi/WabiSabi/Backend/Rounds/Arena.Partial.cs

```diff
@@ -26,7 +26,7 @@ public async Task<InputRegistrationResponse> RegisterInputAsync(InputRegistratio
 		}
 		catch (Exception ex) when (IsUserCheating(ex))
 		{
-			Prison.Ban(request.Input, request.RoundId);
+			Prison.CheatingDetected(request.Input, request.RoundId);
```

Comment:
No, it means that the input you are trying to register is spent. It can be that your wallet is broken, not fully-synchronized or you are just trying to cheat.